.vscode/settings.json

```diff
@@ -1,4 +1,5 @@
 {
+  "editor.defaultFormatter": "esbenp.preter-vscode",
```

Comment:
esbenp.prettier-vscode

Reply:
Oh god, no idea how that happened, haha 😅 
```suggestion
  "editor.defaultFormatter": "esbenp.prettier-vscode",
```